sdk/endpoints/batch/.net/Contracts/BatchScoringInputProperties.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace BatchInferencingSamples
+{
+    internal class BatchScoringInputProperties
+    {
+        public Dictionary<string, BatchScoringInput> InputData { get; set; } = new Dictionary<string, BatchScoringInput>();
```

Comment:
Could you expose this as interface?

Reply:
These are contract classes, I don't think interfaces makes sense in this case..!